acceptance/tests/resource/service/ticket_5024_systemd_enabling_masked_service.rb

```diff
@@ -0,0 +1,47 @@
+test_name 'Systemd masked services are unmasked before attempting to start'
+
+# This test in intended to ensure that a service which was previously marked
+# as masked and then set to enabled will first be unmasked.
+# Confine to RedHat 7, which uses the systemd provider.
+confine :to, :platform => 'el-7'
+confine :to, {}, hosts.select { |host| host['template'] =~ /redhat/ }
```

Comment:
I decided to confine against redhat 7, as it's one of a handul of platforms with systemd as its default service provider. Is checking the template name crazy? It seems to be the only element that has the string 'redhat.' We can't confine against 'el-7' since systemd is not CentOS's default.